returns/contrib/hypothesis/laws.py

```diff
@@ -28,13 +28,15 @@ class _Settings(NamedTuple):
 
     settings_kwargs: Dict[str, Any]
     use_init: bool
+    mark_xfail: bool
 
 
 def check_all_laws(
     container_type: Type[Lawful],
     *,
     settings_kwargs: Optional[Dict[str, Any]] = None,
     use_init: bool = False,
+    mark_xfail: bool = False,
```

Comment:
Nope, sorry.

I was expecting that `xfail` should be a mark of the whole test module, like so:

```python
pytestmark = pytest.mark.xfail(...)

check_all_laws(...)
```

Reply:
Oh, now I got it hahaha